packages/maskbook/src/plugins/RedPacket/types.ts

```diff
@@ -33,7 +33,7 @@ export interface RedPacketAvailability {
     total: string
     claimed: string
     expired: boolean
-    ifclaimed: boolean
+    claimed_amounts: number
```

Comment:
Will it break the current code? Does this object published as a payload or stored in the database?

Reply:
Neither of them, actually `ifclaimed` has not been used before. And this object will be calculated every time when load the red packet to decide the status.